### PR TITLE
Portfolio Project: Make desktop version

### PR DESCRIPTION
**Pull request by alexpuente0**
## In this pull request we worked on:

- Adding media queries to make the site responsive to screen size
- Updated HTML code to show/hide certain information depending on the screen size
- Style the site so it looks more user-friendly in desktop form.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,21 +30,39 @@
...
-      </p>
+      <div class="winfo">
+        <h1 class="title">Hey There. I’m Alex!</h1>
+        <h3 class="subtitle">I am a Software Developer</h3>
+        <p class="paragraph">
+          I can help you build a product, feature or website. Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
```
Comment (moise10r) [inline index.html:65]: - [x] I really appreciate your design, you did great work so far, however, please implement these few changes and you will be good to go, see this image below, your `padding-right` value of your `welcomeinfo` section in the desktop version is not the same as on the original design.




- [ IT SHOULD LOOK LIKE THIS INSTEAD]

File: index.html [hunk 2]
```diff
@@ -103,98 +121,115 @@ <h2 class="ptitle t2">My Recent Works</h2>
             src="./icons/mainproject.png"
             alt="My Latest Project"
           />
-          <h2 class="ptitle t1">Multi-Post Stories</h2>
```
Comment (moise10r) [inline index.html:106]: - [x] The My Recent Works title should be in the same line with this vertical line see :arrow_down: 

- your design 




- Figma design

**Review by moise10r — CHANGES_REQUESTED** [review]
### Hi **@alexpuente0**,

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!


### To highlight 💪🏻

- No linter errors. ✔️ 
- Github flow was correctly used.  ✔️   
- Good commit messages ✔️ 
- Descriptive README file.  ✔️ 
 👍  Good job!



## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

- [x] Your website presents some overlap kindly fix this.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [ OPTIONAL] What if you add a screenshot of your portfolio to the README file to make more descriptive

Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @moise10r when you comment so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (alexpuente0) [reply index.html:65]: thank you for the feedback!
i just adjusted the padding and also the background image to give it more space between elements.

File: index.html (hunk 2 above)
Comment (alexpuente0) [reply index.html:106]: I solved this issue as well, thank you!

**Pull request merged**